core/src/main/scala/org/apache/spark/sql/TiContext.scala

```diff
@@ -71,61 +71,6 @@ class TiContext(val sparkSession: SparkSession) extends Serializable with Loggin
   val autoLoad: Boolean =
     conf.getBoolean(TiConfigConst.ENABLE_AUTO_LOAD_STATISTICS, defaultValue = true)
 
-  // tidbMapTable does not do any check any meta information
```

Comment:
<img width="867" alt="image" src="https://user-images.githubusercontent.com/17768378/158726074-090e699c-fe32-4d5b-af6d-ad4d0cee9859.png">
Can you double-check it

Reply:
The doc is outdated, only TiSpark >=2.0 and <2.3 need tidbMapDatabase. I will delete this Note for we don't support TiSpark before 2.3